modules/default/clock/clock.js

```diff
@@ -188,7 +188,7 @@ Module.register("clock",{
 				moonSet = nextMoonTimes.set;
 			}
 			const isVisible = now.isBetween(moonRise, moonSet) || moonTimes.alwaysUp === true;
-			const illuminatedFractionString = moonIllumination.fraction.toFixed(2) * 100 + '%';
+			const illuminatedFractionString = Math.round(moonIllumination.fraction * 100) + '%';
```

Comment:
when page refresh and Math.round(moonIllumination.fraction * 100) =  0.526781678199768 or something between 0 and 1
```
clock.js:208 Uncaught (in promise) ReferenceError: illuminatzedFractionString is not defined
    at Class.getDom (clock.js:208)
    at main.js:118
    at new Promise (<anonymous>)
    at updateDom (main.js:117)
    at Object.updateDom (main.js:558)
    at Class.updateDom (module.js:366)
    at notificationTimer (clock.js:49)
```
illuminatedFractionString = Math.floor(moonIllumination.fraction * 100) resolve this issue